.gitignore

```diff
@@ -1,5 +1,11 @@
 *~
-topologies/kne/testbed.kne.yml
 .vscode/
 # used by `make validate_paths`
-openconfig_public/
\ No newline at end of file
+openconfig_public/
+launch.json
+settings.json
+topologies/kne/*.yml
```

Comment:
Revert to topologies/kne/testbed.kne.yml?

Reply:
I think so...initially I had multiple files based on the test, but now we can have just one I think